lib/fog/sakuracloud/models/compute/servers.rb

```diff
@@ -44,10 +44,12 @@ def create_and_attach_volume(server, options)
             Fog::Logger.warning("Create Volume")
             sakuracloud_api_token        = options[:sakuracloud_api_token] || Fog.credentials[:sakuracloud_api_token]
             sakuracloud_api_token_secret = options[:sakuracloud_api_token_secret] || Fog.credentials[:sakuracloud_api_token_secret]
+            size_mb = options[:volume][:size_mb].to_i || 20480
```

Comment:
I referred official API document.
http://developer.sakura.ad.jp/cloud/api/1.1/disk/

Unfortunately, it is written in Japanese.
Following JSON is the source I referred.

```
{
  "Disk":{
    "ID":"XXXXXXXXXXXXX",
    "Name":"New Disk",
    "Description":"Creating an empty disk",
    "Connection":"virtio",
    "ConnectionOrder":null,
    "ReinstallCount":0,
    "Availability":"available",
    "SizeMB":20480,
    "MigratedMB":null,
    "WaitingJobCount":null,
```